src/renderers/dom/client/ReactDOMIDOperations.js

```diff
@@ -91,14 +90,14 @@ var ReactDOMIDOperations = {
    * @param {string} name A property name to remove, see `DOMProperty`.
    * @internal
    */
-  deletePropertyByID: function(id, name, value) {
+  deletePropertyByID: function(id, name) {
```

Comment:
Hey, *you* asked *me* for this review, just doing my job. You didn't think I'd just stamp this did you? 😝

Now if you hadn't changed this method at all, it probably would have slipped past.